test/JDBC/expected/temp_oid.out

```diff
@@ -3,7 +3,7 @@ show babelfishpg_tsql.temp_oid_buffer_size
 GO
 ~~START~~
 text
-0
+65536
```

Comment:
where do we have any tests for this?

Reply:
I have been manually testing to validate that there is no lock contention, using the following test:

session 1:
```
begin tran
go
create table #t1 (a int)
go
```
session 2:
```
begin tran
go
create table #t1 (a int)
go
```
Without the engine changes enabled, the second session will hang until the first session either commits or aborts its transaction. With the engine enabled, the second session will successfully create the table.

Reply:
Added a test in JDBCTempTable.java which does the above actions. I confirmed that the test fails when the hooks above are disabled, and succeeds when they are enabled.

Reply:
can we have some tests based on pg catalog queries which cal validate locking assumption. Same for rel caches.